modyn/model_storage/internal/storage_strategies/full_model_strategies/pytorch_full_model.py

```diff
@@ -0,0 +1,21 @@
+import pathlib
+
+import torch
+from modyn.model_storage.internal.storage_strategies.full_model_strategies.abstract_full_model_strategy import (
+    AbstractFullModelStrategy,
+)
+
+
+class PyTorchFullModel(AbstractFullModelStrategy):
+    """
+    This full model strategy naively stores the whole model on disk (default pytorch implementation).
+    """
+
+    def _store_model(self, model_state: dict, file_path: pathlib.Path) -> None:
+        torch.save(model_state, file_path)
+
+    def _load_model(self, base_model_state: dict, file_path: pathlib.Path) -> None:
+        base_model_state.update(torch.load(file_path))
```

Comment:
Since we are overwriting the base state, I would stick to this solution. We use this approach in all of our strategies in order to not materialize several model state dictionaries.